contracts/RiskFund/RiskFund.sol

```diff
@@ -143,18 +153,20 @@ contract RiskFund is
     }
 
     /**
-     * @notice Swap array of pool assets into base asset's tokens of at least a mininum amount.
+     * @notice Swap array of pool assets into base asset's tokens of at least a minimum amount
      * @param markets Array of vTokens whose assets to swap for base asset
-     * @param amountsOutMin Minimum amount to recieve for swap
-     * @return Number of swapped tokens.
+     * @param amountsOutMin Minimum amount to receive for swap
+     * @param paths A path conststing of PCS token pairs for each swap
+     * @return Number of swapped tokens
      */
     function swapPoolsAssets(
         address[] calldata markets,
         uint256[] calldata amountsOutMin,
         address[][] calldata paths
     ) external override returns (uint256) {
         _checkAccessAllowed("swapPoolsAssets(address[],uint256[],address[][])");
-        require(poolRegistry != address(0), "Risk fund: Invalid pool registry.");
+        address poolRegistry_ = poolRegistry;
+        require(poolRegistry_ != address(0), "Risk fund: Invalid pool registry");
```

Comment:
Yeah, I figured it would be better if we show a more detailed error message here since we're checking the storage var and not a param. But if we have natspec, I'd say people can look up the meaning in the doc.

Fixed in ff8cad2